### ISSUE TITLE
Lagging behind in updates

### ISSUE THREAD

**Issue by ViperCode7** [Yepoleb/gogdb#25]
Is there a way to force or request an update of a game?

I'm seeing right now Sniper Ghost Warrior Contracts 2 got out with ver 1.01 but as of this writing the gogdb page is still at 1.0.
And I've also seen some other games in the past having their entries fall behind on gogdb while Steam versions kept getting updated.

Some kind of a button with a captcha would be a good addition to any game page.

**Comment by Yepoleb** [comment]
GOG DB updates every 2 hours, I think that's often enough and all updates should be picked up. A manual update button might seem simple, but the updater is not written in a way that allows single game updates easily. Sorry, this won't be added.

**Comment by ViperCode7** [comment]
I see... however, while you're right that 2 hours is fairly good update frequency, I see the update system isn't functioning properly.
Previously I've kept a game's name out of this report, which I know also to have been updated but it's still not showing up on gogdb. On that game I noticed the update about 20 hours ago, at the time of writing this message, and on gogdb it's still showing an old version.
